source/tutorials/nix-language.md

```diff
@@ -1676,29 +1676,29 @@ It may produce a different hash or even a different package version.
 
 A derivation's output path is fully determined by its inputs, which in this case come from *some* version of Nixpkgs.
 
-This is why we recommend to [avoid search paths](search-path), except in examples intended for illustration only.
+This is why we recommend [avoiding search paths](search-path), except in examples intended for illustration only.
 :::
 
 <details><summary>Detailed explanation</summary>
 
-The example imports the Nix expression from the search path `<nixpkgs>`, and applies the resulting function to an empty attribute set `{}`.
+The example imports the Nix expression from the search path `<nixpkgs>` and applies the resulting function to an empty attribute set `{}`.
```

Comment:
This is a sequence, not parallel. I'm not super strong on English grammar and placing commas, but removing that one feels it would distort the meaning.